server/index.js

```diff
@@ -10,6 +10,7 @@ import * as constant from '../client/js/constant';
 import Board from '../client/js/board.js';
 
 var b = new Board();
+let pendingGame = false;
```

Comment:
`inProgressGame`. "pending" suggests this variable means everyone is waiting for a game